<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Segment_train_dataset</class>
 <widget class="QWidget" name="Segment_train_dataset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Creating dataset</string>
  </property>
  <widget class="QPushButton" name="pushButton_creat_segment_dataset">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>生成数据集</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择数据文件夹：</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_select_inputdata_dir">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="segment_inputdata_path">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>311</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
